lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -92,6 +92,82 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
                                                        operand, vApproximate);
         return success();
       });
+  patterns.onOp(
+      "GridSample", 20,
+      [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Torch::ValueTensorType resultType;
+        Value input;
+        Value grid;
+        std::string mode;
+        Value interpolationMode;
+        std::string padding;
+        Value paddingMode;
+        int64_t align;
+        Value alignCorners;
+
+        if (binder.tensorOperandAtIndex(input, 0) ||
+            binder.tensorOperandAtIndex(grid, 1) ||
+            binder.tensorResultType(resultType))
+          return failure();
```

Comment:
Include a `rewriter.notifyMatchFailure()` just so we get better debug messages.

Reply:
yes sounds better anyway